src/components/ProfileCard/ProfileCard.constants.ts

```diff
@@ -0,0 +1,97 @@
+import EmojiEventsIcon from '@mui/icons-material/EmojiEvents';
+import PersonIcon from '@mui/icons-material/Person';
+import SchoolIcon from '@mui/icons-material/School';
+import StarIcon from '@mui/icons-material/Star';
+
+import type { ProfileCardSize, ProfileCardVariant, ProfileStats } from './ProfileCard.types';
+
+export const PROFILE_CARD_SIZES: Record<ProfileCardSize, ProfileCardSize> = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+export const PROFILE_CARD_VARIANTS: Record<ProfileCardVariant, ProfileCardVariant> = {
+  default: 'default',
+  compact: 'compact',
+  expanded: 'expanded',
+} as const;
+
+export const SIZE_CONFIGS = {
+  small: {
+    avatarSize: 48,
+    padding: 16,
```

Comment:
SIZE_CONFIGS.padding is a pixel value (16). Styles currently divide by 8 to convert px to spacing units which is brittle and confusing.

**Sugerencia:**
```typescript
Store padding as spacing units directly (numbers) and avoid ad-hoc conversions. Example:

small: { paddingUnits: 2 } // theme.spacing(2) => 16px if spacing = 8

Then in styles use `theme.spacing(sizeConfig.paddingUnits)`.
```